### DISCUSSION TITLE
Providing Real-World Sensor Data to CARLA

### DISCUSSION THREAD

**Discussion by tasawor** [carla-simulator/carla#6226] [General]
Hi, i would like to ask that is there any provision by which we can input data from real world (let's say camera) in the CARLA. i found somewhere that it can be done through ROS bridge, but i don't know how (and if it's even true). Also, can we re-create real world scene, captured by camera, inside CARLA?

**Comment by tmnku** [comment]
Maybe you have to provide a little more information on what you're trying to achieve but in general I would say what you're doing doesn't really make sense in my eyes because you would have to recreate everything that is happening in the camera images inside CARLA so the sensor observation would match the surrounding world which is really hard to do. If you want to recreate the static world from camera images you could try to create a meshed and textured model by using some photogrammetry software. COLMAP is a free and popular choice. You could then try to import that model into the simulation.
